### PR TITLE
WITP : Fix mvp one

### LINKED ISSUE #115: Footer Space in bottom (Contact Page)
image

### LINKED ISSUE #117: CSS Issue in Upcoming Event Page
image

### LINKED ISSUE #119: Header FOSS Logo not redirect to Home Page

### LINKED ISSUE #121: Upcoming Event Mobile Responsive Error
image

### LINKED ISSUE #122: Contact Us Page UI Issue
image

### LINKED ISSUE #127: Become a Member button in Home Page not linked
Become a Member button not linked to any form

### PR DESCRIPTION
**Pull request by ebonynon**
- [X] Fixed #115 but need to center the image, @Shehanka can you help me?
- [X] Fixed #117 
- [X] Fixed #119 
- [X] Fixed #127
- [X]  Fixed #121
- [X] Fixed #122

### CODE REVIEW

**Comment by Shehanka** [comment]
@ebonynon show me a screenshot

**Review by Shehanka — APPROVED** [review]

**Comment by ebonynon** [comment]
Screenshot_2021-01-28_16-28-52

**Comment by ebonynon** [comment]
Screenshot_2021-01-28_17-07-30

**Comment by ebonynon** [comment]
Screenshot_2021-01-28_17-09-16

**Comment by Shehanka** [comment]
@ebonynon Have a look
https://stackoverflow.com/questions/16438173/center-text-in-fixed-height-div

File: src/app/components/contact/contact.component.css [hunk 1]
```diff
@@ -30,6 +30,15 @@ p {
   font-family: 'Baloo Thambi 2', sans-serif;
   text-align: center;
 }
+
+.cat-cover {
+  margin: 50px;
+}
```
Comment (Shehanka) [inline src/app/components/contact/contact.component.css:36]: change class name

File: src/app/components/events/events.component.css [hunk 2]
```diff
@@ -8,3 +8,9 @@
   text-align: justify;
   padding-top: 5rem;
 }
+
+.shib-inu {
+  margin-left: auto;
+  height: 350px;
+  width: auto;
+}
```
Comment (Shehanka) [inline src/app/components/events/events.component.css:16]: provide meaningful name

**Review by Shehanka — CHANGES_REQUESTED** [review]

**Review by Shehanka — APPROVED** [review]
@ebonynon My heart is overflowing with gratitude. Thanks from the bottom of my heart!

**Comment by ebonynon** [comment]
> @ebonynon My heart is overflowing with gratitude. Thanks from the bottom of my heart!

My pleasure. Glad to contribute.

**Pull request merged**